mmv1/third_party/terraform/services/gkeonprem/resource_gkeonprem_bare_metal_cluster_test.go.erb

```diff
@@ -381,6 +393,12 @@ func testAccGkeonpremBareMetalCluster_bareMetalClusterUpdateManualLb(context map
         }
       }
     }
+    binary_authorization {
+      evaluation_mode = "DISABLED"
```

Comment:
Can you please update this mode to `PROJECT_SINGLETON_POLICY_ENFORCE` to test the update of this field from `DISABLED`?

Reply:
Updated. Thanks for the suggestion!